### ISSUE TITLE
problem with Gaussian kernel

### ISSUE THREAD

**Issue by markqzw** [amalF/Kervolution#2]
thanks for your code! but when I use the Gaussian kernel, the loss curve is always the same. do not change. can I ask why? I used my own dataset and the other kernel works well.

**Comment by amalF** [comment]
Hello markqzw,

The training loss of the Gaussian kernel on the MNIST dataset  is decreasing (see image below). Are you able to reproduce the same result as me ? 
![image](https://user-images.githubusercontent.com/4411276/79387562-89c90880-7f31-11ea-8380-b1b8d3f214a7.png)

**Comment by markqzw** [comment]
hello amalF,
thank you very much for your reply. I use my own dataset, the input shape is (5,20,135,91) and the labels is (5,20,135,3) a regression problem. I set 100 epochs. the loss is always the same.

**Comment by amalF** [comment]
Hello,
sorry for the delay
It's going to be hard for me to debug if you are using your own dataset. Did you try different hyperparemeters ?

**Comment by amalF** [comment]
I will close this issue for now. please feel free to reopen 
thanks
